src/Icons/doc/index.rst

```diff
@@ -16,6 +16,18 @@ Installation
 
     $ composer require symfony/ux-icons
 
+If you plan to use provided icon sets, make sure that you have the HTTP client installed:
+
+.. code-block:: terminal
+
+    $ composer require symfony/http-client
+
+If you plan to use the optional HTML syntax, also install the ux-twig-component:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-twig-component
+
```

Comment:
This is already in admonition block the two times HTML syntax is introduced, I don't think this is necessary.

Reply:
is there a way we can fail with a message when this is used without the dependency? if we can, i think mentioning it only where the tags are explained would be enough.

Reply:
> is there a way we can fail with a message when this is used without the dependency

I don't believe so.